src/chocolatey/infrastructure.app/services/NugetService.cs

```diff
@@ -1301,6 +1327,30 @@ public virtual void remove_rollback_directory_if_exists(string packageName)
 
                         foreach (SourcePackageDependencyInfo packageDependencyInfo in resolvedPackages)
                         {
+                            if (packageDependencyInfo is null)
+                            {
+                                ResultMessage message = null;
+
+                                if (config.IgnoreDependencies)
+                                {
+                                    message = new ResultMessage(ResultType.Error, "Unable to resolve dependency chain. This may be caused by a parent package depending on this package, try specifying a specific version to use or don't ignore any dependencies!");
+                                }
+                                else
+                                {
+                                    message = new ResultMessage(ResultType.Error, "An unknown failures happened during the resolving of the dependency chain!");
+                                }
+
+                                 var existingPackage = packageResultsToReturn.GetOrAdd(packageName, (key) =>
+                                 {
+                                    // In general, this value should already be set. But just in case
+                                    // it isn't we create a new package result.
+                                    return new PackageResult(availablePackage, string.Empty);
+                                 });
+                                existingPackage.Messages.Add(message);

```

Comment:
This line, or perhaps the section of code above it, doesn't seem to be lined up correctly.

Reply:
It was the previous lines, I have fixed those up to be in line.